github/Organization.py

```diff
@@ -1326,6 +1328,35 @@ def get_dependabot_alerts(
             url_parameters,
         )
 
+    def get_all_custom_properties(self) -> PaginatedList[OrganizationCustomProperty]:
```

Comment:
shall we rename this?
```suggestion
    def get_custom_properties(self) -> PaginatedList[OrganizationCustomProperty]:
```